airflow-core/src/airflow/ui/src/i18n/locales/nl/dags.json

```
@@ -0,0 +1,55 @@
+{
+  "actions": {
+    "delete": "Verwijder Dag",
+    "deleteDagWarning": "Dit zal alle metadata gerelateerd aan de DAG verwijderen, inclusief DAG Runs en Tasks.",
```

For now I just copied the casing of `DAG` from the English locale file, but should this be `Dag`?

---

We agreed in the community (if you want to know the proof I'd need to search in devlist... but it is there) that from Airflow 3 onwards the term "Dag" is always with an upper case "D" and lower case "ag". (In the past=DAG==the acrony for Directed Acyclic Graph".

So: "Dag" is the corect one.
```suggestion
    "deleteDagWarning": "Dit zal alle metadata gerelateerd aan de Dag verwijderen, inclusief Dag Runs en Tasks.",
```